index.js

```diff
@@ -39,10 +37,18 @@ const Secret = secret => {
 
 const ExpiresAt = t.Integer;
 const Algorithm = t.enums.of(['HS256', 'HS384', 'HS512', 'RS256'], 'Algorithm');
-const UserId = t.Any;
-const RefreshToken = t.String;
-const Token = t.String;
+const UserId = t.String;
 const Prolong = t.Boolean;
+const AccessTokenType = 'AT';
+const RefreshTokenType = 'RT';
+const RefreshTokenSignOptions = {
```

Comment:
Should nbt be added?

Reply:
Pros: Lower refresh requests, prevents a kind of DOS?
Cons: Harder for clients to use